metadata.json

```diff
@@ -42,22 +42,13 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7",
```

Comment:
Technically RHEL 7 isn't really EOL. You can still buy ELS for another 4 years: https://www.redhat.com/en/blog/announcing-4-years-extended-life-cycle-support-els-red-hat-enterprise-linux-7

Whether we want to support that is a valid question.

Reply:
We may offer paid support for RHEL7 in Voxpupuli modules then as well :-D